### PR TITLE
Extract increase coverage

### PR DESCRIPTION
**Pull request by Leon-JS02**
- Added tests for more GOG functions.

### CODE REVIEW

File: web_scraping/gog_scraper/scrape_gog.py [hunk 1]
```diff
@@ -11,22 +11,16 @@
 
 def get_game_urls_from_page(page_html: str) -> list[str]:
     """Given a page, returns a list of all game URLs in that page."""
-
```
Comment (paulauus) [inline web_scraping/gog_scraper/scrape_gog.py:14]: Hashtag formatting.

**Review by paulauus — APPROVED** [review]

File: web_scraping/gog_scraper/scrape_gog_game.py
Comment (MichaelHassan) [inline web_scraping/gog_scraper/scrape_gog_game.py:1]: good job formatting, looks more compact

**Review by MichaelHassan — DISMISSED** [review]
nice testing more of the script.

**Comment by Leon-JS02** [comment]
Mehrge conflict, reapprovals pls

File: web_scraping/gog_scraper/scrape_gog_game.py
Comment (MichaelHassan) [inline web_scraping/gog_scraper/scrape_gog_game.py:1]: I like the better formatting, easier to read.

**Review by MichaelHassan — APPROVED** [review]
Nice increase in tests, more confident in the robustness of the code.

**Review by paulauus — APPROVED** [review]

**Pull request merged**